### ISSUE TITLE
ETag cache tracking

### ISSUE THREAD

**Issue by stoically** [stoically/temporary-containers#394]
As [reported on reddit](https://www.reddit.com/r/firefox/comments/ei05bt/firefox_containers_and_etag_tracking/), containers are susceptible to ETag cache tracking, even though [they shouldn't](https://wiki.mozilla.org/Security/Contextual_Identity_Project/Containers#What_is_.28and_isn.27t.29_separated_between_Containers). Easily reproducible [here](https://lucb1e.com/rp/cookielesscookies/). Not sure if an Bugzilla ticket already exists - might be worth filing one if not.

To mitigate right now using an Add-on, @claustromaniac's [ETag Stoppa](https://addons.mozilla.org/firefox/addon/etag-stoppa/) works well for that.

**Comment by crssi** [comment]
Are you sure? I do not have the same results as you.
I might use something else to mitigate, but I am 99% sure that I rely just on TC for that... before TC I was using ETag Stoppa.

Will double check on a plain vanilla profile with TC only and report back today,

Cheers

**Comment by crssi** [comment]
Done testing.
You are right, but AFAIK this was not the case in the past.
I am sure I had done same tests before ditching ETag Stoppa, which was in this case duplicate.

Thank you for warning, getting ETag Stoppa back into profile right now. 😄 

Cheer

**Comment by stoically** [comment]
Yeah, it's weird that containers don't cover that, must be a bug. I guess I could add the ETag Stoppa functionality as advanced preference into TC as well :thinking:

**Comment by stoically** [comment]
[ClearURLs](https://addons.mozilla.org/firefox/addon/clearurls/) filters ETag headers by default as well.

**Comment by stoically** [comment]
Should be fixed by Firefox 85+

**Comment by crssi** [comment]
Hmm... FF ver 85.
If I open `https://lucb1e.com/rp/cookielesscookies/` in one container and set some value.
Then I open the same link in another container, reload twice and I get the value entered in the first container.

But using ETag Stoppa extension everything is OK.

**Comment by stoically** [comment]
Hm, true. So they missed Etag in their attempt to fix the supercookie problem? oof
